src/main/java/com/zendesk/maxwell/MaxwellConfig.java

```diff
@@ -116,6 +126,10 @@ protected OptionParser buildOptionParser() {
 
 		parser.accepts( "__separator_4" );
 
+		parser.accepts( "httppost_endpoint", "provide an endpoint to send data to using the httppost producer" ).withRequiredArg();
+		parser.accepts( "httppost_interceptor", "optional values include 'hmac', 'basicauth'" ).withRequiredArg();
+		parser.accepts( "httppost_authkey", "secret alias (keyId) for calculating HMAC digest or Username for Basic Auth for httppost producer" ).withRequiredArg();
+		parser.accepts( "httppost_authsecret", "secret key for calculating HMAC digest or password for Basic Auth for httppost producer" ).withRequiredArg();
```

Comment:
A little odd that basic auth and HMAC signing share these options.  I think maybe just have 3 options:

- http_hmac_key
- http_hmac_secret
- http_basic_auth

where http_basic_auth is just "username:password"